nin/backend/generate/TemplateShaderLayer.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @constructor
+ */
+function TemplateLayer(config) {
+  this.config = config;
```

Comment:
It's called config in all our layers and passed in as such in the constructor.
I think it's fair then to call it config.